frontend/app/features/kanban/components/TaskFilter.tsx

```diff
@@ -0,0 +1,139 @@
+              onBlur={handleFilterChange}
+            >
+              <option value="ALL">All</option>
+              <option value="TODO">TODO</option>
+              <option value="IN_PROGRESS">IN PROGRESS</option>
+              <option value="DONE">DONE</option>
+            </select>
+          </div>
+        </div>
+
+        <div className="column is-one-fifth">
+          <label className="label">Filter by Created By:</label>
+          <div className="select is-link">
+            <select
+              aria-label="Filter by Created By"
+              value={createdBy}
+              onChange={(e) => setCreatedBy(e.target.value)}
+              onBlur={handleFilterChange}
+            >
+              <option value="ALL">All</option>
+              {uniqueCreators.map((creator) => (
+                <option key={creator} value={creator}>
+                  {creator}
+                </option>
+              ))}
+            </select>
+          </div>
+        </div>
+
+        <div className="column is-one-fifth">
+          <label className="label">Filter by Assigned To:</label>
+          <div className="select is-info">
+            <select
+              aria-label="Filter by Assigned To"
+              value={assignedTo}
+              onChange={(e) => setAssignedTo(e.target.value)}
+              onBlur={handleFilterChange}
+            >
+              <option value="ALL">All</option>
+              {uniqueAssignees.map((assignee) => (
+                <option key={assignee} value={assignee}>
+                  {assignee}
+                </option>
+              ))}
+            </select>
+          </div>
+        </div>
+
+        <div className="column is-one-fifth">
+          <label className="label">Filter by Due Date:</label>
+          <div className="select is-danger">
+            <select
+              aria-label="Filter by Due Date"
+              value={dueDate}
+              onChange={(e) => setDueDate(e.target.value)}
+              onBlur={handleFilterChange}
+            >
+              <option value="ALL">All</option>
+              {uniqueDueDates.map((date) => (
+                <option key={date} value={date}>
+                  {date}
+                </option>
+              ))}
+            </select>
+          </div>
+        </div>
+
+        <div className="column is-one-fifth">
+          <label className="label">Filter by Created At:</label>
+          <div className="select is-success">
+            <select
+              aria-label="Filter by Created At"
+              value={createdAt}
+              onChange={(e) => setCreatedAt(e.target.value)}
+              onBlur={handleFilterChange}
+            >
+              <option value="ALL">All</option>
+              {uniqueCreatedAtDates.map((date) => (
+                <option key={date} value={date}>
+                  {date}
+                </option>
+              ))}
+            </select>
```

Comment:
no need for this filter, lets keep it  simple, status, assign to and created by filters are enough, please remove others